src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import impl.OperationStrategyImpl;
+import impl.ParseServiceImpl;
+import impl.ReadServiceImpl;
+import impl.ReportServiceImpl;
+import impl.WriteServiceImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import service.OperationStrategy;
+import service.ParseService;
+import service.ReadService;
+import service.ReportService;
+import service.WriteService;
+import strategy.BalanceHandler;
+import strategy.OperationHandler;
+import strategy.PurchaseHandler;
+import strategy.ReturnHandler;
+import strategy.SupplyHandler;
+
+public class Main {
+    private static final String INPUT_FILE_PATH = "src/main/java/database.csv";
+    private static final String REPORT_FILE_PATH = "src/main/java/report.csv";
```

Comment:
It's better to use the appropriate directory for resources files. Change the REPORT_FILE_PATH to 'src/main/resources/report.csv'

Reply:
done.